### DISCUSSION TITLE
Improving continuous events performance (lodash)

### DISCUSSION THREAD

**Discussion by Mil4n0r** [dxc-technology/halstack-react#1782] [Ideas]
I have realized that some of the components (for example Tabs), use event listeners for handling continuous events (scroll, resizing...)

In those scenarios, it's often considered a good practice to debounce or throttle the event handler, preventing it from firing too frequently, especially if the handler involves expensive computations or updates to the DOM. 

This can help improve performance and responsiveness.

I've taken a look at some options, and the consensus seems to be to use [lodash](https://www.npmjs.com/package/lodash?activeTab=readme), a collection of utility functions that don't require any additional libraries or dependencies.

**Comment by Jialecl** [comment]
I think that before adding Lodash, we should know how much improvement can we get and if it is worth it.
